### ISSUE TITLE
Bug: Accessibility Issue with TOC toggle, theme, and jump

### ISSUE THREAD

**Issue by jenstrickland** [w3c/sustainableweb-wsg#159]
## Description
While starting the accessibility review for the horizontal review, using the FAST Checklist, I noticed that the utility in the lower left corner read "link up arrow" without the context for what the link is, as there is a known issue where `aria-hidden="true"` does not work with VoiceOver and Chrome. I tested VoiceOver with Safari and it did respect `aria-hidden="true"`.

There's also an empty radio group announced for the theme chooser.

## Technology Used
- Chrome Version 140.0.7339.215
- VoiceOver on Mac 15.6.1 (24G90)

## Recommendation

- [ ] Remove the `aria-hidden="true"` and let the symbol be read associated with the link text.

- [ ] For the theme toggle, remove the span for `aria-hidden="true"` and replace `title="theme toggle icon"` with alt text.

## Current Code

```
<p id="toc-nav">
  <a id="toc-jump" href="#toc" aria-labelledby="toc-jump-text"><span aria-hidden="true">↑</span> <span id="toc-jump-text">Jump to Table of Contents</span></a>

<a id="toc-toggle" href="#toc" aria-labelledby="toc-collapse-text"><span aria-hidden="true">←</span> <span id="toc-collapse-text">Collapse Sidebar</span></a>

<a id="toc-theme-toggle" role="radiogroup" aria-label="Select a color scheme">
          <span aria-hidden="true"><img src="https://www.w3.org/StyleSheets/TR/2021/logos/dark.svg" title="theme toggle icon"></span>
          <span>
          <label>
          <input name="color-scheme" type="radio" value="light" checked="">
          <span>light</span>
        </label><label>
          <input name="color-scheme" type="radio" value="dark">
          <span>dark</span>
        </label><label>
          <input name="color-scheme" type="radio" value="auto">
          <span>auto</span>
        </label>
          </span>
        </a>
</p>
```

### empty radio group
<img width="1279" height="923" alt="empty radio group screenshot" src="https://github.com/user-attachments/assets/12ca5283-afea-4890-bc74-8a1d1bde8a6a" />

### link up arrow

<img width="1311" height="970" alt="link up arrow screenshot" src="https://github.com/user-attachments/assets/f9adeabf-00a4-4e36-95ca-e9170594298b" />
